mkosi/resources/mkosi.md

```diff
@@ -1133,6 +1133,11 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
   separated list of package specifications. This option may be used
   multiple times in which case the specified package lists are combined.
 
+`InitrdHostonlyMicrocode=`, `--initrd-hostonly-microcode=`
```

Comment:
Needs to be updated. Also I'd go for `MicrocodeHost=` after all because it is nicely consistent with `KernelModulesIncludeHost=`

Reply:
Ah, forgot about that one. Changed the name as well, although I think it's a bit less clear I have no strong preference.